test/expected/create_hypertable.out

```diff
@@ -37,6 +37,7 @@ select create_hypertable('test_schema.test_1dim', 'time');
  test_schema | test_table | table | postgres
 (2 rows)
 
+insert into test_schema.test_1dim VALUES ('2004-10-19 10:23:54+02',1.0);
```

Comment:
Yep, Travis is failing because of this. Once fixed, you have my approval to merge.